modals/ideas.js

```diff
@@ -0,0 +1,21 @@
+const { v4 } = require('uuid');
+
+const { getIdeas, setIdeas } = require('./utils/storage.js');
```

Comment:
Do you use these functions anywhere in this file?

Reply:
Not in this function. you're right. 
Regarding the name of directory, I remember, that we named it the same way in the prev. project.

Reply:
The name `modals` makes no sense here. Did you mean `models`?

Reply:
Yes, indeed. Fixed.